src/main/java/team/themoment/hellogsmv3/domain/oneseo/service/ModifyRealOneseoArrivedYnService.java

```diff
@@ -34,10 +36,36 @@ private Oneseo switchRealOneseoArrivedYn(Oneseo oneseo) {
         return Oneseo.builder()
                 .id(oneseo.getId())
                 .member(oneseo.getMember())
-                .desiredMajors(oneseo.getDesiredMajors())
                 .oneseoSubmitCode(oneseo.getOneseoSubmitCode())
+                .desiredMajors(oneseo.getDesiredMajors())
                 .realOneseoArrivedYn(oneseo.getRealOneseoArrivedYn() == YesNo.YES ? YesNo.NO : YesNo.YES)
+                .finalSubmittedYn(oneseo.getFinalSubmittedYn())
+                .wantedScreening(oneseo.getWantedScreening())
                 .appliedScreening(oneseo.getAppliedScreening())
                 .build();
     }
+
+    private void assignSubmitCode(Oneseo oneseo) {
+        Integer maxSubmitCodeNumber = oneseoRepository.findMaxSubmitCodeByScreening(oneseo.getAppliedScreening());
+        int newSubmitCodeNumber = (maxSubmitCodeNumber != null ? maxSubmitCodeNumber : 0) + 1;
+
+        String submitCode;
+        ScreeningCategory screeningCategory = oneseo.getAppliedScreening().getScreeningCategory();
+        switch (screeningCategory) {
+            case GENERAL -> {
+                submitCode = "A-" + newSubmitCodeNumber;
+            }
+            case SPECIAL -> {
+                submitCode = "B-" + newSubmitCodeNumber;
+            }
+            case EXTRA -> {
+                submitCode = "C-" + newSubmitCodeNumber;
+            }
+            default -> {
+                throw new IllegalArgumentException("Unexpected value: " + screeningCategory);
+            }
+        }
```

Comment:
```suggestion
        switch (screeningCategory) {
            case GENERAL -> submitCode = "A-" + newSubmitCodeNumber;
            case SPECIAL -> submitCode = "B-" + newSubmitCodeNumber;
            case EXTRA -> submitCode = "C-" + newSubmitCodeNumber;
            default -> throw new IllegalArgumentException("Unexpected value: " + screeningCategory);
        }
```

중괄호 생략이 더 가독성이 높을것 같습니다!

Reply:
네네 훨씬 좋네요.
감사합니다